testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/model/UserConsentModelTest.java

```diff
@@ -320,6 +320,10 @@ public void deleteClientScopeTest(KeycloakSession session) {
 
             ClientScopeModel fooScope = KeycloakModelUtils.getClientScopeByName(realm, "foo");
             realm.removeClientScope(fooScope.getId());
+            ClientScopeModel barScope = KeycloakModelUtils.getClientScopeByName(realm, "bar");
+            try {
+                realm.removeClientScope(barScope.getId());
```

Comment:
If it is expected that clientScope cannot be removed, then it can be good to add `fail` into the `try` block. As otherwise it is not tested that removing of client scope failed. So something like:
```
try {
    realm.removeClientScope(barScope.getId());
    Assert.fail("It is not expected to successfully remove client scope bar");
} catch (Exception expected) {}
```
```

Reply:
I replaced it with `Assert.assertTrue(realm.removeClientScope(barScope.getId()));`

as `removeClientScope` returns a boolean

Reply:
@kaustubh-rh Thanks, but if I understand correctly, since removing of the client scope is still OK and returns true, it means that you are not testing at all the new functionality, which you added to `ClientScopeResource` ? The test changes in `UserConsentModelTest.java` were passing even before your changes if I understand correctly.

To test that it is not possible to remove all realm client scopes, I suggest to add a new test method to `ClientScopeTest`, which tests admin REST API. You can probably inspire from some of the tests in this class.

Reply:
Hi @mposolda , thanks, I did the required changes.